source/version_helper.js

```diff
@@ -10,5 +11,34 @@ module.exports = {
       // in that case we fallback to a branch version
       return semver.coerce(version === '0.0.0-development' ? branchVersion : version);
   },
+  CockroachVersionGreaterThan21_1: async function(connection) {
+    const versionRow = await connection.query("SELECT version() AS version", { type: QueryTypes.SELECT });
```

Comment:
hmm now i wonder, is there a way we can just load the version info once on startup, and save that string as an instance variable? then these functions can use that variable instead of re-sending the query each time? if this seems too tricky then no need to do it now

Reply:
I think we need to query in the telemetry case since each Sequelize object could theoretically be a different version of CRDB.

I updated how we look for the CRDB version for testing however by grabbing in from the environment vars.
PTAL

Reply:
hmm right. so i guess there's no existing hook anywhere for "when the connection gets initialized" ?